.circleci/config.yml

```diff
@@ -29,6 +29,9 @@ jobs:
     docker:
       - image: circleci/openjdk:8-jdk
     working_directory: /tmp/workspace
+    environment:
+      # TODO (landrito): make this a project wide environment variable.
```

Comment:
https://circleci.com/gh/googleapis/gapic-generator/edit#env-vars

Reply:
I decided against doing project wide env variable since the showcase version will be changing rapidly until API is fully fleshed out.